CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+## Contributing to Axelar Examples
+
+We appreciate your interest in contributing to Axelar Examples! This document `CONTRIBUTING.md` provides a high-level overview of how you can get involved.
+
+### Table of Contents
+  * [Introduction](#introduction)
+  * [Getting Started](#getting-started)
+  * [How to Contribute](#how-to-contribute)
+  * [Code of Conduct](#code-of-conduct)
+
+### Introduction
+
+`Axelar Examples` is a collection of examples and tutorials for the Axelar Network. Before you start, we encourage you to read the rest of this document, which contains information on how and what you can contribute.
+
+### Getting Started
+
+If you are new to the Axelar Network, we encourage you to read the [Axelar Documentation](https://docs.axelar.network/) to learn more about the Axelar Network and how to get started. 
+
+If you are new to contributing to open-source projects, we encourage you to read [How to Contribute to Open Source](https://opensource.guide/how-to-contribute/).
+
+---
+### How to Contribute
+
+1. Fork the repository and clone it to your local machine.
+2. Install the dependencies for the project as indicated in the [README.md](https://github.com/axelarnetwork/axelar-examples#readme) file.
+3. Create a new branch for your contribution with the following command:
+
+   ```shell
+   git checkout -b my-fix-branch
+   ```
+4. Make your changes in the new git branch 
+5. Commit your changes. Your commit message should follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification.
+
+   ```shell
+   git commit -a
+   ```
+Note: the optional commit `-a' command line option will automatically "add" and "rm" edited files.
```

Comment:
mismatch between backtick (`) and single quote (')

Reply:
Updated